scripts/deploy.sh

```
@@ -0,0 +1,233 @@
+    fi
+    
+    log_info "Application built successfully."
+}
+
+build_docker_image() {
+    log_info "Building Docker image..."
+    
+    cd "$PROJECT_ROOT"
+    
+    # Build image
+    docker build -t context-store-mcp:latest -t context-store-mcp:$(date +%Y%m%d-%H%M%S) .
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)